uwrt_mars_rover_utils/src/uwrt_params_test_node.cpp

```diff
@@ -0,0 +1,42 @@
+#include <ros/console.h>
```

Comment:
should test code go in a separate test folder instead of src?

Reply:
we haven't been doing that so far. we could do something like that if you wanted in the future? Idk if thats what people usually do for ros stuff

Reply:
i guess its fine for now since test files have the word test in them